crates/wasmtime/src/runtime/externals/table.rs

```diff
@@ -111,7 +111,12 @@ impl Table {
             let table = Table::from_wasmtime_table(wasmtime_export, store);
             let wasmtime_table = table.wasmtime_table(store, iter::empty());
             (*wasmtime_table)
-                .fill(store.gc_store_mut()?, 0, init, ty.minimum())
+                .fill(
+                    store.gc_store_mut()?,
+                    0,
+                    init,
+                    ty.minimum().try_into().unwrap(),
```

Comment:
In general, especially in the runtime, we want to avoid `.unwrap()`. To that end could this failure case be turned into an error? 

This I think comes up if a 32-bit host tries to build a table with >= u32::MAX elements.